### ISSUE TITLE
What is the license of this project?

### ISSUE THREAD

**Issue by Gityeeter** [AndroidAudioMods/ViPERFX_RE#10]
I know the Viper4Android app is closed source, but I cannot find any license for ViPERFX_RE. Is ViPERFX_RE open-source, can I use it in my projects? I will give explicit credit and attribution.

**Comment by Martmists-GH** [comment]
I'd like to publish this project under the BSD-2-Clause license, but unfortunately the legality of licensing reverse-engineered code is very much a grey area and we want to avoid any issues regarding this. 

For example, following U.S. Law, section 1201 (part of the DMCA), allows us to reverse engineer ViPERFX for the purpose of interoperability; In other words, we may reverse engineer it to allow recompiling for unsupported devices. 

I am not a lawyer, and as such I cannot guarantee using the effects specified in this project for other programs also counts as interoperability.

That said, I have identified the following (open-source) libraries used in ViPERFX:
- [Bauer stereophonic-to-binaural DSP](https://bs2b.sourceforge.net/)
- [Freeverb](https://ccrma.stanford.edu/~jos/pasp/Freeverb.html)
- [libsndfile](http://www.mega-nerd.com/libsndfile/)

---

TL;DR Feel free to use these however you'd like, although we'd request you to add a link back to this project if you do. We also recommend adding a notice stating that the software includes parts of ViPERFX by ViPER520 (edit: Supposedly KuGou.com now owns it, but I cannot find any confirmation anywhere). We cannot guarantee anything about the legality of using this code in your own software.

**Comment by Gityeeter** [comment]
Thank you! Yes, I will give credit and leave a link to this repository in my project.
